tests/unit/memory_storage_client/test_memory_storage_client.py

```diff
@@ -42,31 +40,44 @@ async def test_write_metadata(tmp_path: Path) -> None:
     )
 
 
-async def test_persist_storage(tmp_path: Path) -> None:
+@pytest.mark.parametrize(
+    ('persist_storage'),
+    [
+        (True),
+        (False),
+    ],
```

Comment:
Maybe a list of tuples?

```suggestion
    [
        (True,),
        (False,),
    ],
```